scripts/vote_tw_csm2.py

```diff
@@ -0,0 +1,654 @@
+    print("set_console_globals(interface=interface)")
+    if TYPE_CHECKING:
+        interface: Any = ...
+
+
+TW_DESCRIPTION = "Proposal to use TW in Lido protocol"
+
+# TWG
+TWG_MAX_EXIT_REQUESTS_LIMIT = 13000
+TWG_EXIT_REQUESTS_PER_FRAME = 1
+TWG_FRAME_DURATION = 48  # in hours
+
+## Oracle consensus versions
+AO_CONSENSUS_VERSION = 4
+VEBO_CONSENSUS_VERSION = 4
+CSM_CONSENSUS_VERSION = 3
+
+EXIT_DAILY_LIMIT = 20
+TW_DAILY_LIMIT = 10
+
+EXIT_EVENTS_LOOKBACK_WINDOW_IN_SLOTS = 7200
+
+NOR_EXIT_DEADLINE_IN_SEC = 30 * 60
+
+NOR_VERSION = ["6", "0", "0"]
+SDVT_VERSION = ["6", "0", "0"]
+
+def encode_staking_router_proxy_update(implementation: str) -> Tuple[str, str]:
+    proxy = interface.OssifiableProxy(contracts.staking_router)
+    return proxy.address, proxy.proxy__upgradeTo.encode_input(implementation)
+
+def get_repo_uri(repo_address: str) -> str:
+    contract = interface.Repo(repo_address).getLatest()
+    return contract["contentURI"]
+
+def encode_proxy_upgrade_to(proxy: Any, implementation: str) -> Tuple[str, str]:
+    proxy = interface.OssifiableProxy(proxy)
+    return proxy.address, proxy.proxy__upgradeTo.encode_input(implementation)
+
+def encode_wv_proxy_upgrade_to(proxy: Any, implementation: str) -> Tuple[str, str]:
+    proxy = interface.WithdrawalContractProxy(proxy)
+
+    return proxy.address, proxy.proxy_upgradeTo.encode_input(implementation, b'')
+
+def submit_proposal(call_script: Sequence[Tuple[str, str]], description: Optional[str] = "") -> Tuple[str, str]:
+    proposal_calldata = []
+
+    for call in call_script:
+        (address, calldata) = call
+        proposal_calldata.append((address, 0, calldata))
+    print("Dual governance address:", contracts.dual_governance.address)
+    return (
+        contracts.dual_governance.address,
+        contracts.dual_governance.submitProposal.encode_input(
+            proposal_calldata, description
+        ),
+    )
+
+def encode_oracle_upgrade_consensus(proxy: Any, consensus_version: int) -> Tuple[str, str]:
+    oracle = interface.BaseOracle(proxy)
+    return oracle.address, oracle.setConsensusVersion.encode_input(consensus_version)
+
+def encode_staking_router_update_csm_module_share() -> Tuple[str, str]:
+    """Encode call to update CSM share limit"""
+    return (
+        contracts.staking_router.address,
+        contracts.staking_router.updateStakingModule.encode_input(
+            CS_MODULE_ID,
+            CS_MODULE_NEW_TARGET_SHARE_BP,
+            CS_MODULE_NEW_PRIORITY_EXIT_THRESHOLD_BP,
+            CS_MODULE_MODULE_FEE_BP,
+            CS_MODULE_TREASURY_FEE_BP,
+            CS_MODULE_MAX_DEPOSITS_PER_BLOCK,
+            CS_MODULE_MIN_DEPOSIT_BLOCK_DISTANCE,
+        )
+    )
+
+def to_percent(bp: int) -> float:
+    """
+    Convert basis points to percentage.
+    """
+    return bp / 10000 * 100
+
+def create_tw_vote(tx_params: Dict[str, str], silent: bool) -> Tuple[int, Optional[Any]]:
+    f"""
```

Comment:
please look at the recent votes in archive folder and add the static description at the beginning of this file - we are deploying this particular vote to hoodi and we need it to be related only to hoodi - you can leave all holesky/mainnet stuff in other branches which we will use for later votes